blackbelt/commands/dep.py

```diff
@@ -13,9 +13,12 @@ def cli():
 
 
 def validate_dep(ctx, param, dep):
+    if os.path.isdir(dep): # check the provided project_dir
+        click.echo('The dependency is a local directory, using whatever is currently in node_modules')
+        if not os.path.isdir(os.path.join(dep, 'node_modules')):
+            raise click.BadParameter('There are no node_modules to analyze in the `'+dep+'` project directory')
```

Comment:
Install `flake8` or `pylama` 😉 Although, those would just notify you about the whitespace. Following is a proper solution to the problem:

```python
"There are no node_modules to analyze in the '{}' project directory".format(dep)
```